boat-scaffold/src/test/java/com/backbase/oss/codegen/java/BoatSpringCodeGenTests.java

```diff
@@ -267,7 +267,7 @@ void shouldGenerateValidations(boolean useLombok, boolean bigDecimalsAsStrings)
         var compiler = new MavenProjectCompiler(BoatSpringCodeGenTests.class.getClassLoader());
         var projectDir = new File(output);
         int compilationStatus = compiler.compile(projectDir);
-        assertEquals(0, compilationStatus);
+//        assertEquals(0, compilationStatus);
```

Comment:
Why is it commented out?

Reply:
I've just checked it locally and it works correct i.e. `assertEquals(0, compilationStatus);` is successful